.github/workflows/zeebe-release.yml

```diff
@@ -244,6 +244,50 @@ jobs:
           fi
           shopt -u nocasematch # reset it
           echo "result=${PRE_RELEASE}" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ env.RELEASE_BRANCH }}
+      - name: Install Zeebe changelog tool
+        run: |
+          wget https://github.com/zeebe-io/zeebe-changelog/releases/download/0.7.0/zeebe-changelog_0.7.0_Linux_i386.tar.gz
+          tar -xzvf zeebe-changelog_0.7.0_Linux_i386.tar.gz
+          chmod +x zcl
+      - name: Generate Changelog for patch release
+        id: gen-changelog
+        run: |
+          changelog="Release ${{ inputs.releaseVersion }}"
+          # Right now we only support patch release for generating the change log
+          # as it is the easiest to automate and the most repetitive work to do on a release
+          if [[ "$version" =~ ^8\.[1-9][0-9]*\.[1-9][0-9]*$ ]]
+          then
+            # Next, to add the release labels to release's issues, specifying the previous and current release in place of ZCL_FROM_REV and ZCL_TARGET_REV, respectively.
+            #
+            # ZCL_TARGET_REV should be replaced with the tag name for the version you are releasing, and 
+            # ZCL_FROM_REV should be replaced as the tag name for the previous version, based on the release type:
+            #
+            # PATCH: the tag for the previous patch version on the same minor branch. e.g. if you're releasing 1.2.3, then ZCL_FROM_REV=1.2.2.
+            # To find the previous patch version we extract the patch version and subsctract by one
+            patchVersion=$(echo ${{ inputs.releaseVersion }} | sed 's/8\.[1-9][0-9]\*\.//')
+            majorMinor=$(echo ${{ inputs.releaseVersion }} | sed 's/\.[1-9][0-9]\*$//')
+            ZCL_FROM_REV="$majorMinor".$(( patchVersion - 1 ))
+            ZCL_TARGET_REV="${{ inputs.releaseVersion }}"
+
+            # This will add labels to the issues in GitHub. You can verify this step by looking at closed issues. They should now be tagged with the release.
+            ./zcl add-labels \
+            --token=${{ secrets.GITHUB_TOKEN }} \
```

Comment:
💭 I expect to have enough permissions to update the issues, as we are also able to create a release. Per default it should be ok ? https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Did you try it?

Reply:
Yes it is working now see https://github.com/camunda/zeebe/actions/workflows/camunda-platform-release-manual.yml?query=branch%3AZelldon-patch-1